pkg/controller/resourcegraphdefinition/controller.go

```diff
@@ -38,6 +38,10 @@ import (
 	"github.com/kubernetes-sigs/kro/pkg/metadata"
 )
 
+// +kubebuilder:rbac:groups=kro.run,resources=resourcegraphdefinitions,verbs=get;list;watch;create;update;patch;delete
+// +kubebuilder:rbac:groups=kro.run,resources=resourcegraphdefinitions/status,verbs=get;update;patch
+// +kubebuilder:rbac:groups=kro.run,resources=resourcegraphdefinitions/finalizers,verbs=update
+
```

Comment:
This also got removed in #819 we should not re-introduce it unless we re-introduce a working rbac generation